### DISCUSSION TITLE
skywalking service instance not showing

### DISCUSSION THREAD

**Discussion by kannan84240** [apache/skywalking#9050] [Q&A]
skywalking service instance ip address and UUID not showing automatically, so i cant analyze jvm, cpu usage etc, I'm monitoring tomcat and java application only
![instance_not_showing](https://user-images.githubusercontent.com/73342985/168065011-14051b7e-8235-4be3-9c16-bb40e0e26f68.png)

**Comment by wu-sheng** [comment]
I am not sure, the demo and our env are working well.

**Reply by kannan84240** [reply]
thanks for your reply, Its working fine version 8 showing all menus and metrics but version 9 only not working, kindly share any documents for this issue.

**Reply by wu-sheng** [reply]
I don't think there is a known issue. You could check demo.skywalking.apache.org (skywalking/skywalking), they are all good.

**Reply by kannan84240** [reply]
Okay, i ll check again, can you plz share step by step configuration for both server and agent, because in website I cant understand that's why i'm asking. thank you.

**Reply by wu-sheng** [reply]
There is no configuration or complex steps. the demo is hosting codes of the showcase, https://skywalking.apache.org/docs/skywalking-showcase/latest/readme/

**Reply by kannan84240** [reply]
can't find any useful information regarding instance not showing, kindly help on this. and (#183) this conversation they are discussed generate UUID manually that's why i'm asking.
